### ISSUE TITLE
Accessibility (major): ensure skip to search properly links to search [argo-000]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/argo#4159]
Within the skip links `<nav>`, the "Skip to search" link refers to `<input id="search_field">`, but the screen reader cannot follow this link to the `<input>` element.

<img width="613" alt="Screenshot 2023-09-13 at 1 50 00 PM" src="https://github.com/sul-dlss/argo/assets/2294288/1ee940b1-96a0-4a76-b5d6-b42507130757">

Notes: https://docs.google.com/spreadsheets/d/1m3obE89_mBCZpRkWAjJAXEr2ZfrUirzjvsJWqYH-G6Y/edit#gid=0
